### DISCUSSION TITLE
can i use `wsldl` on an Arm based machine?

### DISCUSSION THREAD

**Discussion by azinsharaf** [yuk7/wsldl#146] [Q&A] [answered]
can i use `wsldl` on an Arm based machine to create a Arch Linux distro in WSL 1? As you know `ArchWSL` doesn't support arm based architecture.

**Comment by yuk7** [comment, marked as answer]
Yes, wsldl supports Arm64 architecture.
However you must use it with Arm64 linux system image.

**Comment by azinsharaf** [comment]
thank you, @yuk7 ! 
I was able to install it successfully on WSL1 (on a Win 11 VM hosted on apple M1 Max). I put the steps here in case someone else needs it. 

![image](https://github.com/yuk7/wsldl/assets/10101468/96df9b3a-cfff-438f-9506-02bf7bbd7b23)
